bot.py

```diff
@@ -470,18 +476,20 @@ async def notify_if_voice_client_is_missing(self, message):
             return True
         return False
 
-    async def play(self, message, command_content):
+    async def play(self, message, command_content, deque_append_left=False):
```

Comment:
```suggestion
    async def play(self, message, command_content, prepend=False):
```
I think somethink more succinct is in order.

Reply:
I think `prepend` is more descriptive when describing how you are playing. Actually, I think there has to be an even better word describing that we want to play the given play next. Maybe `playnext`, `immediate`, `first`?

In any case, `deque_append_left` isn't descriptive of what the parameter _does_ in the "play whatever is in `command_content`" sense.

Reply:
Imo `prepend` is less descriptive, but I can get behind `playnext` 👍